hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/HazelcastClientCacheManager.java

```diff
@@ -47,7 +44,6 @@
 
     private final HazelcastClientInstanceImpl client;
     private final ClientCacheProxyFactory clientCacheProxyFactory;
-    private final ConcurrentMap<String, CacheConfig> configs = new ConcurrentHashMap<String, CacheConfig>();
```

Comment:
This map duplicates whatever is already available in `ClientCacheProxyFactory.configs`